straxen/plugins/peaklets/peaklet_classification_som.py

```diff
@@ -27,31 +27,31 @@ class PeakletClassificationSOM(PeakletClassification):
     analysis.
     """
 
-    __version__ = "0.0.1"
-
-    depends_on = "peaklets"
-
-    provides = ("peaklet_classification", "som_peaklet_data")
-    data_kind = dict(peaklet_classification="peaklets", som_peaklet_data="peaklets")
-
-    parallel = True
+    __version__ = "0.2.0"
+    child_plugin = True
 
     som_files = straxen.URLConfig(
         default="resource://xedocs://som_classifiers?attr=value&version=v1&run_id=045000&fmt=npy"
     )
 
+    use_som_as_default = straxen.URLConfig(
```

Comment:
Add option for the future to default back to classic classification if required.